cnf-certification-test/networking/netutil/netutil.go

```diff
@@ -77,3 +77,13 @@ func GetListeningPorts(cut *provider.Container) (map[PortInfo]bool, error) {
 
 	return parseListeningPorts(outStr)
 }
+
+func GetSSHDaemonPort(cut *provider.Container) (string, error) {
+	const findSSHDaemonPort = "grep -w 'Port' /etc/ssh/sshd_config"
+	outStr, errStr, err := crclient.ExecCommandContainerNSEnter(findSSHDaemonPort, cut)
+	if err != nil || errStr != "" {
+		return "", fmt.Errorf("failed to execute command %s on %s, err: %v", findSSHDaemonPort, cut, err)
+	}
+	portNumber := strings.Fields(outStr)[1]
```

Comment:
Should there be a check for length to make sure it has enough elements first?  Just as a safeguard.

Reply:
I can put a condition of min 2 length, because in the config file really it could be anything, like Port 2023 # changed by Banashri